lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -590,6 +590,7 @@ def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
                 return RuleRecordTable()
             else:
                 usage = self._get_usage_from_flow_stat(response.body)
+                self._delete_old_flows(usage.values())
```

Comment:
actually to not block the return can you do 
       ` self.loop.call_soon_threadsafe(self._delete_old_flows(usage.values()))`